### ISSUE TITLE
Add ability to query the entire database

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#3243]
This would ignore scoping and all other functions.

This may be a duplicate as I could not find the original issue, so please close if so.

**Comment by maxxxxxdlp** [comment]
I don't think we have an issue for this, but I mentioned this issue in https://github.com/specify/specify7/issues/2675#issuecomment-1367487823

This issue is a blocker for #2675

**Comment by melton-jason** [comment]
Duplicate of #2903
